src/printer/code_printer.cpp

```diff
@@ -110,5 +114,19 @@ void CodePrinter::pop_block(const std::string_view& suffix, std::size_t num_newl
     add_newline(num_newlines);
 }
 
+void CodePrinter::blame() {
+    if (current_line == blame_line) {
+        *result << std::flush;
+
+        std::cout << "\n\n== Blame =======================================================\n";
+
+        backward::StackTrace st;
+        st.load_here(32);
+        backward::Printer p;
+        p.print(st, std::cout);
```

Comment:
I'm not convinced we want to bake this in unconditionally... I would rather add a build option, and only if enabled we add backward and compile in the blame option. This seems to me like a feature that is mostly useful to developers and less to users (yes, you could argue that this way you can ask the user to just send you the "blame" when encountering an issue, but you can just as well easily reproduce this locally).